Gemfile

```diff
@@ -36,6 +36,9 @@ gem 'bcrypt', '~> 3.1.7'
 # validators
 gem 'email_address'
 
+# nested form
+gem 'cocoon'
```

Comment:
bir gün sağlam bir front-end'cimiz olunca bu bağımlılıktan da kurtulacağız. cocoon en gıcık olduğum bağımlılıklardan :) ama mecburen kullanıyoruz. hatta birisi de benim gibi düşünüp [sormuş](https://softwarerecs.stackexchange.com/questions/43552/cocoon-nested-form-gem-rails-alternative) ama yanıt alamamış. vueJS ile bu sorunu çok temiz çözenler var, gelecekteki front-end'cimize not düşeyim.